src/index.scss

```diff
@@ -31,7 +31,12 @@ body {
 
   .clickable {
     cursor: pointer;
+    
+    .disabled {
```

Comment:
Are you sure you don't mean `&.disabled`?

Reply:
you're right!

Reply:
ah no. the disabled is applied to childs of .clickable :)

Reply:
Ah ok, I trust you :)